deploy/helm/metrics-server-overrides.yaml

```diff
@@ -0,0 +1,183 @@
+# This file is auto-generated.
```

Comment:
@vsinghal13  was this missed before? Should be OK to check in with this unrelated PR though?

Reply:
The file was added in the autoscaler PR.

Reply:
In this PR, the overrides section seems to be covering all the config?